src/PowerShellEditorServices/Services/Template/TemplateService.cs

```diff
@@ -71,41 +70,36 @@ public async Task<bool> ImportPlasterIfInstalledAsync()
                     .AddCommand("Select-Object")
                     .AddParameter("First", 1);
 
-                this.logger.LogTrace("Checking if Plaster is installed...");
+                this._logger.LogTrace("Checking if Plaster is installed...");
 
-                var getResult =
-                    await this.powerShellContext.ExecuteCommandAsync<PSObject>(
-                        psCommand, false, false).ConfigureAwait(false);
+                PSObject moduleObject = (await _executionService.ExecutePSCommandAsync<PSObject>(psCommand, new PowerShellExecutionOptions(), CancellationToken.None).ConfigureAwait(false)).First();
```

Comment:
For future fixing: we need to consolidate our importing of modules we ship (e.g. PSSA, Plaster, and PSReadLine).